web/src/main/kotlin/no/nav/su/se/bakover/web/routes/revurdering/Revurderingsfeilresponser.kt

```diff
@@ -44,7 +44,17 @@ internal object Revurderingsfeilresponser {
 
     val feilutbetalingStøttesIkke = InternalServerError.errorJson(
         "Feilutbetalinger støttes ikke",
-        "feilutbetalinger_støttes_ikke"
+        "feilutbetalinger_støttes_ikke",
+    )
+
+    val fantIngenVedtakSomKanRevurderes = NotFound.errorJson(
+        "Fant ingen vedtak som kan revurderes for angitt periode",
+        "ingenting_å_revurdere_i_perioden",
+    )
+
+    val tidslinjeForVedtakErIkkeKontinuerlig = InternalServerError.errorJson(
```

Comment:
Jeg tror vi har brukt å sette disse "begrensningene" til BadRequest, men jeg ser argumenter for 500 og

Reply:
Hm ja.. litt vrien. I utgangspunktet er det en guard for å unngå ting vi ikke støtter, samtidig som klienter kan komme seg rundt det ved å endre periode. Føles dog mer som en 500 på nåværende tidspunkt.